src/main/java/org/folio/search/cql/ItemSearchFieldModifier.java

```diff
@@ -0,0 +1,12 @@
+package org.folio.search.cql;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class ItemSearchFieldModifier implements SearchFieldModifier {
+
+  @Override
+  public String modify(String inputField) {
+    return inputField.replace("item.", "items.");
```

Comment:
I think that safer to add a check that `inputField` starts with "item." and only then replace it with "items."